geth/node/utils.go

```diff
@@ -46,3 +52,101 @@ func HaltOnInterruptSignal(nodeManager *NodeManager) {
 	}
 	panic("interrupted!")
 }
+
+// logMessageStat logs stat derived from provided DeliveryState.
+func logMessageStat(state notifications.DeliveryState) {
+	stat := generateMessageStat(state)
+	statdata, err := json.Marshal(stat)
+	if err != nil {
+		log.Info(fmt.Sprintf("%s : JSON MARHSAL ERROR : %+q", params.MessageStatHeader, err))
```

Comment:
Shouldn't it be a warning or error? Why do you use capital letters? To follow the current logging format it should be similar to:
```go
log.Warning("failed to marshal common.MessageStat", "err", err)
```